sensor/common/networkflow/manager/manager_impl_test.go

```diff
@@ -179,7 +193,521 @@ func (suite *NetworkflowManagerTestSuite) TestAddNoOriginator() {
 	h.connectionsSequenceID = sequenceID
 
 	err := h.Process(networkInfoOpen, nowTimestamp, sequenceID)
-	suite.NoError(err)
+	s.NoError(err)
+
+	s.Len(h.endpoints, 1)
+}
+
+//endregion
+
+//region networkFlowManager tests
+
+func (s *NetworkFlowManagerTestSuite) TestEnrichConnection() {
+	mockCtrl := gomock.NewController(s.T())
+	m, mockEntityStore, mockExternalSrc, _ := createManager(mockCtrl)
+	containerID := "id"
+	cases := map[string]struct {
+		connPair                    *connectionPair
+		enrichConnections           map[networkConnIndicator]timestamp.MicroTS
+		expectEntityLookupContainer expectFn
+		expectEntityLookupEndpoint  expectFn
+		expectExternalLookup        expectFn
+		expectedIndicator           *networkConnIndicator
+		expectedConnection          *connection
+		expectedStatus              *connStatus
+	}{
+		"Rotten connection": {
+			connPair:                    createConnectionPair(true, true, false, timestamp.Now().Add(-maxContainerResolutionWaitPeriod*2)),
+			expectEntityLookupContainer: expectEntityLookupContainerHelper(mockEntityStore, 1, clusterentities.ContainerMetadata{}, false),
+			expectedStatus: &connStatus{
+				rotten: true,
+			},
+		},
+		"Incoming External connection no external sources hit": {
```

Comment:
I am guessing that the `no` negates the `hit` - if yes, then maybe let's call it a `miss`.
Next, I think that the `miss` here is the `nil` in `expectExternalLookupHelper`. As hit and miss are mostly use for reading the cache, then maybe we could rename it to something like:

```suggestion
		"Incoming External connection should not result in external lookup": {
```

Reply:
See comment below